app/helpers/application_helper/button/vm_stop.rb

```diff
@@ -1,11 +1,16 @@
 class ApplicationHelper::Button::VmStop < ApplicationHelper::Button::Basic
   needs_record
 
+  def calculate_properties
+    super
+    self['title'] = @record.is_available_now_error_message(:stop) if disabled?
+  end
+
   def skip?
     !@record.is_available?(:stop)
   end
 
-  def disable?
-    @record.is_available_now_error_message(:stop)
+  def disabled?
+    !!@record.is_available_now_error_message(:stop)
```

Comment:
@martinpovolny 

the `is_available?(:stop)` [returns `true` if the action is supported, even if a function is not 'currently' available](https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/availability_mixin.rb#L10-L12).

That means that the action may be supported, but at some point the button needs to be disabled with an error message, even if it is usually available for the `@record`.

That's why `is_available?(:stop)` can't be used here.